nginx.conf

```diff
@@ -37,6 +37,13 @@ http {
         listen 80 default_server;
 
         location / {
+            set $CSP 'default-src \'self\';';
+            set $CSP '${CSP} img-src \'self\' www.google-analytics.com;';
+            set $CSP '${CSP} font-src \'self\' fonts.gstatic.com;';
+            set $CSP '${CSP} style-src \'self\' \'unsafe-inline\' fonts.googleapis.com;';
+            set $CSP '${CSP} script-src \'self\' \'sha256-dumdnmFtmXukp90IByXXb9z1Q4gMA2R+DyJAVrVqxmw=\' www.google-analytics.com;';
+            set $CSP '${CSP} connect-src \'self\' *.jwallet.network *.etherscan.io www.google-analytics.com;';
+            add_header Content-Security-Policy $CSP;
```

Comment:
Setting this option here will break running Jwallet on localhost

Reply:
Discussed and solved in Slack